src/main/kotlin/com/seokjoo/todo/presentation/todo/controller/TodoApiController.kt

```diff
@@ -77,14 +79,17 @@ class TodoApiController(
         servletRequest: HttpServletRequest,
     ): ResponseEntity<TodoResponse> {
         val user = authService.findUser(servletRequest.getBearerToken())
-        val todo = todoService.createTodo(request = request.toTodoServiceRequest(), user).toResponse()
+        val todo = todoService.createTodo(request = request.toTodoCreateServiceRequest(), user).toResponse()
         return ResponseEntity.created(URI.create("/todos/${todo.id}")).body(todo)
     }
 
     @PatchMapping("/todos/{id}")
     @Operation(summary = "특정 id todo 업데이트", description = "id를 이용해 todo 한개를 업데이트 합니다.")
-    fun updateTodo(@PathVariable id: Long, @RequestBody @Validated request: TodoRequest): ResponseEntity<TodoResponse> {
-        val todo = todoService.updateTodo(id = id, request = request.toTodoServiceRequest()).toResponse()
+    fun updateTodo(
+        @PathVariable id: Long,
+        @RequestBody @Validated request: TodoUpdateRequest,
+    ): ResponseEntity<TodoResponse> {
+        val todo = todoService.updateTodo(id = id, request = request.toTodoUpdateServiceRequest()).toResponse()
```

Comment:
```suggestion
        val todo = todoService.updateTodo(id = id, request = request.toUpdateRequest()).toResponse()
```

이것도 매우매우 사소한데, ServiceDto로 변환시키는 메서드의 이름이 좀 긴 것 같아서...! 요런 네이밍은 어떠세여?

Reply:
f03729f636ce50e1bf27402739299322a111a31e 조아요 반영 ~!